schemas/common_definitions.json

```diff
@@ -42,12 +43,25 @@
         "type": "string",
         "description": "The text that is used for the 'Hide guidance' link."
       },
-      "content": {
-        "$ref": "common_definitions.json#/content"
+      "contents": {
+        "$ref": "common_definitions.json#/contents"
       }
     }
   },
   "content": {
+    "description": "Allows customisation of guidance text.",
```

Comment:
This description is wrong. 

I'm not sure this definition should be in common definitions. It's only used in `confirmation.json` and `interstitial.json` and having it here adds to the confusion around `contents`. It may be better to accept a small amount of duplication and put it in both `confirmation.json` and `interstitial.json`.